src/coreclr/src/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -1876,6 +1876,38 @@ internal FieldInfo GetField(RuntimeFieldHandleInternal field)
             return retval;
         }
 
+        internal override Attribute? GetCustomAttribute(Type attributeType, bool inherit)
+        {
+            if (attributeType == null)
+                throw new ArgumentNullException(nameof(attributeType));
+
+            RuntimeType? attributeRuntimeType = attributeType.UnderlyingSystemType as RuntimeType;
+
+            if (attributeRuntimeType == null)
+                throw new ArgumentException(SR.Arg_MustBeType, nameof(attributeType));
+
+            return CustomAttribute.GetCustomAttribute(this, attributeRuntimeType, inherit);
+        }
+
+        internal override AttributeUsageAttribute GetAttributeUsageAttribute()
+        {
+            if (!this.IsSubclassOf(typeof(Attribute)) && this != typeof(Attribute))
```

Comment:
```suggestion
            if (!this.IsSubclassOf(typeof(Attribute)))
```

System.Attribute is abstract. This also applies elsewhere.